src/dali/plugin/input/csv/blockfi.py

```diff
@@ -111,7 +137,25 @@ def load(self) -> List[AbstractTransaction]:
                             crypto_received=line[self.__AMOUNT_INDEX],
                         )
                     )
-                elif line[self.__TYPE_INDEX] == _WITHDRAWAL:
+                elif entry_type == _ACH_WITHDRAWAL:
+                    last_withdrawal_fee = None
+                    result.append(
+                        OutTransaction(
+                            plugin=self.__BLOCKFI,
+                            unique_id=Keyword.UNKNOWN.value,
+                            raw_data=raw_data,
+                            timestamp=f"{line[self.__TIMESTAMP_INDEX]} -00:00",
+                            asset=line[self.__CURRENCY_INDEX],
+                            exchange=self.__BLOCKFI,
+                            holder=self.account_holder,
+                            transaction_type="Sell",
+                            spot_price=Keyword.UNKNOWN.value,
```

Comment:
Do they have spot price in the trade file? They used to have it in the transaction file up until last year, but then they removed it recently. I asked their customer support if they could put it back, and they basically said: "No"... 

Hopefully they'll release a REST API at some point (they have been saying it's in their roadmap for at least one year).